src/main/java/com/gtbr/vaultbot/model/PrivateChannel.java

```diff
@@ -0,0 +1,32 @@
+package com.gtbr.vaultbot.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Table
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PrivateChannel {
+
+    @Id
+    @Column
+    private String privateChannelId;
+
+    @OneToOne
+    @JoinColumn(name = "guild_id")
```

Comment:
Property name not necessary because hibernate generate the name in correct pattern, just use this porperty in joincolumns.